validator/client/src/main/java/tech/pegasys/teku/validator/client/restapi/apis/PostKeys.java

```diff
@@ -43,6 +43,7 @@ public PostKeys(final KeyManager keyManager) {
             .operationId("ImportKeystores")
             .summary("Import Keystores")
             .description("Import keystores generated by the Eth2.0 deposit CLI tooling.")
+            .security("bearerAuth")
```

Comment:
Maybe we should have a constant for `bearerAuth` since we wind up having to specify it in a bunch of places.  Or even just a `.securityBearerAuth` or (`.withBearerAuthSecurity()`) method.